etor/src/main/java/gov/hhs/cdc/trustedintermediary/external/hapi/HapiOrderConverter.java

```diff
@@ -152,13 +152,12 @@ private MessageHeader createMessageHeader() {
 
         messageHeader.setEvent(OML_CODING);
 
-        messageHeader.setMeta(
-                new Meta()
-                        .addTag(
-                                new Coding(
-                                        "http://terminology.hl7.org/CodeSystem/v2-0103",
-                                        "P",
-                                        "Production")));
+        var meta = new Meta();
+
+        // Adding processing id of 'P'
+        meta.addTag("http://terminology.hl7.org/CodeSystem/v2-0103", "P", "Production");
```

Comment:
Do we always want this to be P? I thought this was a flag for controlling if messages were for prod or test

Reply:
Good question. Side note, I only added the comment, the actual code for adding `P` has been there for a while. Initially, this was needed, but I realized that the messages have `P` when coming from RS. This raises a new issue:

If the message is for testing, it will have `T`, or it might include another code like `D`. If this is the case, this code will change that to a `P`. We need to see if this is the behavior that we want. I will add an entry in the engineering channel regarding this.